services/web/server/src/simcore_service_webserver/conversations/_socketio.py

```diff
@@ -62,16 +97,79 @@ async def _send_message_to_recipients(
 ):
     async for _ in limited_as_completed(
         (
-            send_message_to_user(
-                app, recipient, notification_message, ignore_queue=True
-            )
+            send_message_to_user(app, recipient, notification_message)
             for recipient in recipients
         ),
         limit=_MAX_CONCURRENT_SENDS,
     ):
         ...
 
 
+async def notify_conversation_created(
+    app: web.Application,
+    *,
+    recipients: set[UserID],
+    project_id: ProjectID,
+    conversation: ConversationGetDB,
+) -> None:
+    notification_message = SocketMessageDict(
+        event_type=SOCKET_IO_CONVERSATION_CREATED_EVENT,
+        data={
+            **ConversationCreatedOrUpdatedEvent(
+                project_id=project_id,
+                **conversation.model_dump(),
+            ).model_dump(mode="json", by_alias=True),
```

Comment:
TIP: Check constant `RESPONS_MODEL_POLICY` https://github.com/itisfoundation/osparc-simcore/blob/5732a12e07e63d5ce55010ede9b9ab543bb9b278/packages/service-library/src/servicelib/rest_constants.py

Reply:
Yup, but this constant belongs to `rest_constants`. We should move it out from there, or create a new one.